### DISCUSSION TITLE
mermaid not worked?

### DISCUSSION THREAD

**Discussion by amwps290** [cotes2020/jekyll-theme-chirpy#1972] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I used mermaid to draw the graphic and found that the graphic doesn't show up on the web page, it's still the same code as before, here is my code

```mermaid
flowchart LR
  A(["sqlite init"]) --> B(["创建 sqlite_master"]) --> C(["读取 sqlite_master"]) --> D(["加载数据到内存"])
```
![image](https://github.com/user-attachments/assets/5d23c4eb-99a9-47e4-8aae-731f119205b0)

### Operations you have already tried

I tested it in mermaid live editor and it works fine, is there something wrong?

![image](https://github.com/user-attachments/assets/acfb37b6-0fc0-455c-8051-17166530b0f1)

**Comment by amwps290** [comment]
It's available on github, but not on jeklly.

**Comment by huanyushi** [comment]
You don't offer a full code for your page. But this is OK in the original `jekyll-theme-chirpy` file:

![image](https://github.com/user-attachments/assets/c8baf363-606d-46a1-b78f-9e242ac0689e)

**Comment by huanyushi** [comment, marked as answer]
I think you should read the [document](https://chirpy.cotes.page/posts/write-a-new-post/#mermaid):
![image](https://github.com/user-attachments/assets/d4403460-b3f0-433b-92b1-2339d111855c)

**Reply by amwps290** [reply]
Thanks, I really didn't notice this, I thought it would be as straightforward as markdown. No other configuration options needed
